contrib/inventory/azure_rm.py

```diff
@@ -812,6 +897,12 @@ def _load_settings(self):
                 except:
                     pass
 
+            if not config.has_section('cache'):
+                config.add_section('cache')
```

Comment:
I think you can simply skip it if config file doesn't have `cache` section. I assume this is for backward compatibility.

Reply:
We get an error if the section doesn't exist. The original code is doing a *try catch*, what I don't like